FirebaseSessions/Sources/Settings/SettingsCacheClient.swift

```diff
@@ -15,6 +15,12 @@
 
 import Foundation
 
+#if SWIFT_PACKAGE
+  @_implementationOnly import GoogleUtilities_UserDefaults
+#else
+  @_implementationOnly import GoogleUtilities
+#endif // SWIFT_PACKAGE
```

Comment:
CocoaPods here?

Reply:
Like this?: 
```suggestion
#endif // COCOAPODS
```

Reply:
Yeah, I had this for rollouts change, double check if this is the general commenting way.

Reply:
Ok, understood. I think the practice is to add a comment that corresponds to the opening `#if`, so it's easier to trace clauses when there are many nested `#if`s.

Reply:
I see, it's good then. I should probably change the rollouts instead. Gonna resolve the comment!